src/braket/aws/aws_device.py

```diff
@@ -296,6 +298,10 @@ def _default_shots(self):
             AwsDevice.DEFAULT_SHOTS_QPU if "qpu" in self.arn else AwsDevice.DEFAULT_SHOTS_SIMULATOR
         )
 
+    @property
+    def _default_max_parallel(self):
+        return AwsDevice.DEFAULT_MAX_PARALLEL
```

Comment:
Just making sure, this is to future-proof for device-specific defaults, right?

Reply:
Yup.